docs/authorization.md

```diff
@@ -164,6 +164,69 @@ parent workspace.
 
 Service accounts declared within a workspace don't have access to initializing workspaces.
 
+## API binding authorizer
+
+If the requested resource type is part of an API binding, then the API binding authorizer verifies that
+the request is not exceeding the maximum permission policy of the related API export.
+Currently, the "local policy" maximum permission policy type is supported.
+
+### Local policy
+
+The local maximum permission policy delegates the decision to the RBAC of the related API export workspace.
+
+Example:
+
+Given an API binding for type `foo` declared in workspace `consumer` that refers to an API export declared in workspace `provider`
+and a user `user-1` having the group `group-1` requesting a `create` of `foo` in the `default` namespace in the `consumer` workspace,
+the API binding authorizer verifies that `user-1` is allowed to execute this request by delegating to `provider`'s RBAC using prefixed attributes.
+
+Here, the API binding authorizer prepends the `apis.kcp.dev:binding:` prefix to the username and all groups the user belongs to.
+Using prefixed attributes prevents RBAC collisions i.e. if `user-1` is granted to execute requests within the `provider` workspace directly.
+
+For the given example RBAC request looks as follows:
+
+- Username: `apis.kcp.dev:binding:user-1`
+- Group: `apis.kcp.dev:binding:group-1`
+- Resource: `foo`
+- Namespace: `default`
+- Workspace: `provider`
+- Verb: `create`
+
+The following role and role binding declared within the `provider` workspace will grant access to the request:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: foo-creator
+  clusterName: provider
+rules:
+- apiGroups:
+  - foo.api
+  resources:
+  - foos
+  verbs:
+  - create
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: user-1-foo-creator
+  namespace: default
+  clusterName: provider
+subjects:
+- kind: User
+  name: apis.kcp.dev:binding:user-1
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: foo-creator
+```
+
+Note: The same authorization scheme is enforced when executing the request via the virtual API Export API server.
```

Comment:
```suggestion
Note: The same authorization scheme is enforced when executing the request of a claimed resource (... link permission claims ...) via the virtual API Export API server, i.e. a claimed resource is bound to the same maximal permission policy. Only the actual owner of that resources can go beyond that policy.
```

Reply:
> `... link permission claims ...`

we don't have a markdown for this 😞 that's next on my list but adding the rest.